packages/client/src/internal/client/methods.ts

```diff
@@ -253,9 +258,46 @@ export class ClientMethods extends ClientCore implements IClientMethods {
         };
     }
 
-    public async deposit(params: any): Promise<any> {
-        //TODO : 토큰 입금
-        return params;
+    public async deposit(email: string, amount: number): Promise<ContractTransaction[]> {
```

Comment:
```suggestion
    public async deposit(email: string, amount: number): Promise<ContractTransaction[]> {
```

amount를 BigNumber로 처리하게 새로운 이슈를 생성해주세요